Add missing properties from rec.ttl

metesaka opened the pull request:
This commit targets to solve #673

gtfierro commented:
Looks like a great start! Please do the same for *all* of the ICT equipment classes in REC (you should be able to find them at https://dev.realestatecore.io/ontology/), and make sure the properties themselves are also defined in the recpatches.ttl file

metesaka commented:
I added missing properties for all children (and grandchildren) of ict_equipment and also fixed some deprecation messages. However I did not understand what you mean by defining the properties, they're not explicitly defined under rec.ttl either.

gtfierro reviewed (approved):
Looks great, thank you!
